.github/workflows/iroha2-custom-image.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         with:
           ref: ${{ github.event.inputs.CHECKOUT_REF }}
       - name: Build wasm libs
-        run: ./scripts/build_wasm.sh --profile=${{ env.IROHA2_PROFILE }} --target=libs
+        run: ./scripts/build_wasm.sh --target=libs
```

Comment:
this was good as it was, why change?

Reply:
likewise for other files in CI

Reply:
1. `--target=libs` is easier to validate since only named arguments are used in `build_wasm.sh`
2. `--profile=${{ env.IROHA2_PROFILE }}` works for `deploy`, but for `PROFILE=profiling`, it will require changes in memory, fuel, and timeouts. I think `deploy` should be used everywhere by default.

Reply:
2. ok, but how is someone to build a custom image with profile `profiling` now?